src/test/java/racing/RacingServiceTest.java

```diff
@@ -19,9 +20,11 @@ public class RacingServiceTest {
     @Test
     @DisplayName("자동차가 1칸 앞으로 전진한다.")
     void 자동차_전진_테스트() throws CarLocationException {
-        Car car = new Car("hbae", new Location(0));
+        Integer location = 0;
+        Integer moveAmount = 1;
+        Car car = new Car("hbae", new Location(location));
         Location newLocation = car.moveForward();
-        assertThat(newLocation.value()).isEqualTo(1);
+        assertThat(newLocation.value()).isEqualTo(location + moveAmount);
```

Comment:
테스트 코드에 비즈니스 코드를 넣지않는편이 좋다고 생각합니다.
검증부는 더욱 로직을 넣는것은 불필요하다고 생각해요.

왜냐면, 테스트 코드는 특정 메소드를 테스트 할 뿐입니다.
- "테스트 코드는 시스템이 예상대로 동작하는지 확인하는 데 중점을 둬야 합니다" 
- "테스트 코드는 간결하고 명확해야 합니다."


```suggestion
        assertThat(newLocation.value()).isEqualTo(1);
```

- 객체간의 비교
`assertThat(car.moveForward()).isEqualTo(new Location(1))`

- 자동차를 비교하는 방법
```
car.moveForward();

assertThat(car).isEqualTo(new Car("hbae", 1));
```

객체간의 비교를 하려면 무엇을 구현해야할까요?
객체간의 비교를하면 값을 꺼내는 패턴을 덜 사용하게 되지 않을까요?

Reply:
equals랑 hashCode 오버라이딩을 추가해주면 될 것 같습니다ㅎㅎ
수정해보겠습니다!